universal-application-tool-0.0.1/app/services/applicant/Block.java

```diff
@@ -205,8 +205,9 @@ public boolean isCompletedInProgramWithoutErrors() {
   }
 
   private boolean isCompleteInProgram() {
-    return getQuestions().stream()
-        .anyMatch(question -> question.isAnsweredOrSkippedOptionalInProgram());
+    return getQuestions().size() == 0
```

Comment:
GetQuestions() -> ImmutableLIst which has isEmpty().

Maybe add a javadoc that allows for no questions.

Reply:
done